translator/source_code/cotrip_translator.py

```diff
@@ -0,0 +1,409 @@
+
+import xmltodict
+import json
+from datetime import datetime, timezone
+import uuid
+import random
+import string
+import sys, getopt
+from jsonschema import validate
+from jsonschema import ValidationError
+import logging
+from collections import OrderedDict
+import re
+import dateutil.parser
+
+
+
+help_string = """ 
+
+Usage: python cotrip_translator.py [arguments]
+
+Global options:
+-h, --help                  Print this usage information.
+-i, --input                 specify the xml file to translate
+-o, --output                specify the output file for generated wzdx geojson message """
+
+# Translator
+
+
+def wzdx_creator(messages, info=None, unsupported_message_callback=None):
+    if not messages:
+        return None
+   #verify info obj 
+    if not info:
+        info=initialize_info()
+    if not validate_info(info):
+        return None
+    wzd = {}
+    wzd['road_event_feed_info'] = {}
+    # hardcode
+    wzd['road_event_feed_info']['feed_info_id'] = info['feed_info_id']
+    wzd['road_event_feed_info']['update_date'] = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
+    wzd['road_event_feed_info']['publisher'] = 'CDOT'
+    wzd['road_event_feed_info']['contact_name'] = 'Abinash Konersman'
```

Comment:
These hardcoded values (`contact_name`, `contact_email`, etc) are used later in the file as well, can we make them a variable so we only need to change them once?

Reply:
should be fixed now

Reply:
These values are referenced in only one location now which is good, but Peter is right - we should pull them out to an environment variable or script argument so we can easily change things like contact email in the future.

Reply:
set those as environment variable and set the environment variable in env_var.sh file in translator/source_code so before we run our program we have to run the .sh script file in our local computer then run the program.